utilities/gsheets-import/resultsImporter.py

```diff
@@ -21,21 +21,27 @@
 
 from oauth2client.service_account import ServiceAccountCredentials
 
-GOOGLE_SHEET_URL = ""
-
 SCOPES = ['https://spreadsheets.google.com/feeds',
           'https://www.googleapis.com/auth/drive']
 
+# TEST_STATES = ["Pass", "Warning", "Fail", "Manual", "Not Applicable", "Not Implemented", "Test Disabled",
+#                "Could Not Test"]
+TEST_STATES = ["Pass", "Fail", "Warning", "Not Implemented", "Test Disabled", "Could Not Test", "Manual",
+               "Not Applicable"]
+
 
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("--json", required=True)
+    parser.add_argument("--sheet", required=True)
+    parser.add_argument("--credentials", required=True)
```

Comment:
Minor, but could we add a default argument (and as such remove 'required') to avoid extra command line args when used manually?

Reply:
Yeah, happy to default to 'credentials.json' in cwd if you like.